features/support/env.rb

```diff
@@ -0,0 +1,77 @@
+# IMPORTANT: This file is generated by cucumber-rails - edit at your own peril.
+# It is recommended to regenerate this file in the future when you upgrade to a
+# newer version of cucumber-rails. Consider adding your own code to a new file
+# instead of editing this one. Cucumber will automatically load all features/**/*.rb
+# files.
+
+ENV["RAILS_ENV"] ||= "test"
+ENV["APP_ENV"] ||= "test"
+
+# require 'cucumber/rails'
+require 'capybara/cucumber'
+require 'capybara/mechanize'
+
+# frozen_string_literal: true
+
+Capybara.app_host = "http://127.0.0.1:3000" 
```

Comment:
As discussed, this setup requires the rails server to be running and the test will use that rails server and whatever DB it is pointing at too. This is unusual in the  world of cucumber, unless I am missing something. 

This means it could be running against a local development database (or worse, if you run `bundle exec cucumber` on production 🤔 )

It is not truncating the test data either. I have tested this and it is creating 3 assessments for each run.

```
 # run test rails server instead
  RAILS_ENV=test rails s
 ```
 
We could, instead, be using hooks to run a test rails server on a specific port in a before_suite for example, and using that so that it is 
a. transparently using test database 
b. enable database cleaning to prevent leaking data between tests
c. not using the typical host and port that a dev server might